src/sentry/search/events/builder/discover.py

```diff
@@ -320,7 +320,12 @@ def load_config(
         Mapping[str, Callable[[Direction], OrderBy]],
     ]:
         self.config: DatasetConfig
-        if self.dataset in [Dataset.Discover, Dataset.Transactions, Dataset.Events]:
+        if self.dataset in [
+            Dataset.Discover,
+            Dataset.Transactions,
+            Dataset.Events,
+            Dataset.IssuePlatform,
+        ]:
             self.config = DiscoverDatasetConfig(self)
```

Comment:
I'm a bit surprised this works, but maybe it makes sense since we have the same columns as both events and transactions in this table

Reply:
🤷‍♀️